spec/controllers/api_controller_spec.rb

```diff
@@ -17,4 +17,44 @@
     expect(response_json).to include("docx")
     expect(response_json).to include("json")
   end
+
+  describe "export" do
+    it 'exports single json' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.